deploy/clowdapp.yaml

```diff
@@ -919,3 +919,6 @@ parameters:
   - name: PULP_API_GUNICORN_WORKERS
     description: Number of gunicorn workers in the API pods
     value: "2"
+  - name: PULP_LIST_OF_ADMINS
+    description: List of user accounts to become Pulp admin accounts
+    value: "/O=rhds/OU=serviceaccounts/UID=ddebrito,/O=rhds/OU=serviceaccounts/UID=dkliban,/O=rhds/OU=serviceaccounts/UID=mdepaulo,/O=rhds/OU=serviceaccounts/UID=bmbouter,/O=rhds/OU=serviceaccounts/UID=kchim,edge-api-dev"
```

Comment:
Make the default an empty list.

Reply:
And put the value in app-interface or something?

Reply:
yes, add the value for the stage deployment.